docs/ConfigurationFile.md

```diff
@@ -227,6 +227,7 @@ exports.config = {
         tagExpression: [],  // <string[]> (expression) only execute the features or scenarios with tags matching the expression
         timeout: 20000,     // <number> timeout for step definitions
         ignoreUndefinedDefinitions: false, // <boolean> Enable this config to treat undefined definitions as warnings.
+        scenarioLevelReporter: false // Enable this to make webdriver.io behave as if scenarios and not steps were the tests.
```

Comment:
This property should also be documented in https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cucumber-framework/cucumber-framework.d.ts#L111

Reply:
Right away! ᕕ( ᐛ )ᕗ